src/main/java/org/skillfinder/models/User.java

```diff
@@ -1,46 +1,33 @@
 package org.skillfinder.models;
 
 
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
 
+import java.util.List;
+
+@Data
 @Document(collection = "users")
 public class User {
     @Id
     private String id;
-    private String firstname;
-    private String lastname;
-
-    public User() {
-    }
-
-    public User(String firstname, String lastname) {
-        this.firstname = firstname;
-        this.lastname = lastname;
-    }
-
-    public String getFirstname() {
-        return firstname;
-    }
-
-    public void setFirstname(String firstname) {
-        this.firstname = firstname;
-    }
+    private String name;
+    private String email;
+    private String locality;
```

Comment:
For now it's ok - but do you know better way to represent locality? (E.g for faster search in future)